R/check_description.R

```diff
@@ -250,6 +250,8 @@ Please send a pull request if you need support for this license.",
   if (current_license == "MIT + file LICENSE") {
     author <- this_desc$get_author(role = "cph")
     cph <- paste(c(author$given, author$family), collapse = " ")
+    cph <- gsub("\\(", "\\\\(", cph)
```

Comment:
I think we can make this more generic.

```suggestion
    cph <- gsub("\\", "\\\\", cph)
```

Reply:
I tried it, but it did not work. I came up with an ugly more generic version in [7b73501](https://github.com/inbo/checklist/pull/103/commits/7b7350197ed7f301716838861403e8e041b5769f)